### PR TITLE
added alignment settings

### PR DESCRIPTION
**Pull request by nharbo**

### CODE REVIEW

File: Rye/RyeExample/Rye VC/RyeDemoViewController.swift [hunk 1]
```diff
@@ -56,19 +58,23 @@ class RyeDemoViewController: UITableViewController {
...
-            updateDismissMode(for: indexPath)
-        case .viewType:
-            updateViewMode(for: indexPath)
-        case .position:
-            updatePosition(for: indexPath)
-        case .animationtype:
-            updateAnimationType(for: indexPath)
-        case .ignoreSafeAreas:
-            updateIgnoreSafeAreas(for: indexPath)
+        if let section = Section(rawValue: indexPath.section) {
```
Comment (jakobmygind) [inline Rye/RyeExample/Rye VC/RyeDemoViewController.swift:61]: Why remove the force unwrap tho?

**Review by jakobmygind — APPROVED** [review]
LGTM, maybe fix the formatting and new spaces added a little bit

File: Rye/RyeExample/Rye VC/RyeDemoViewController.swift (hunk 1 above)
Comment (nharbo) [reply Rye/RyeExample/Rye VC/RyeDemoViewController.swift:61]: I had a crash.. Just wanted to make sure it wouldn't happen again :)

Comment (jakobmygind) [reply Rye/RyeExample/Rye VC/RyeDemoViewController.swift:61]: But if this crashes, isn't it a coding error? There's no dynamic data here, and also this is not production code, I'd say we'd rather crash than swallow the error silently here?

Comment (nharbo) [reply Rye/RyeExample/Rye VC/RyeDemoViewController.swift:61]: It's in the demo project, yea. Just thought it was better to click twice than to crash, when you're there just to see the functionality in action. Didn't investigate too much about the crash, but sure, we can revert if that's preferred 😊👍

Comment (nharbo) [reply Rye/RyeExample/Rye VC/RyeDemoViewController.swift]: I agree, that's a bit odd! I'll try and see if I can reproduce it

**Comment by pbodsk** [comment]
Don't forget to update the README with a description of the new parameters 😄

**Pull request merged**